internal/components/setup/kind.go

```diff
@@ -106,6 +106,13 @@ func KindSetup(e2eConfig *config.E2EConfig) error {
 		if err := createKindCluster(kindConfigPath, e2eConfig); err != nil {
 			return err
 		}
+	} else {
+		// export the kubeconfig path for command line
+		err := os.Setenv("KUBECONFIG", kubeConfigPath)
+		if err != nil {
+			return fmt.Errorf("could not export kubeconfig file path, %v", err)
+		}
+		logger.Log.Infof("export KUBECONFIG=%s", kubeConfigPath)
```

Comment:
After the test, shall we unset the env var?

Reply:
What do you think? @kezhenxu94

Reply:
> After the test, shall we unset the env var?

This is to make it convenient running other commands in terminal after e2e command right? If so we might want to unset the env var in `cleanup` command, if possible, we might even want to revert `KUBECONFIG` to the original value before we change it.

Reply:
OK. I thought you want to persist the env var after the e2e exits. LGTM